README.md

```diff
@@ -61,6 +61,14 @@ pipx ensurepath
 sudo pipx ensurepath --global # optional to allow pipx actions with --global argument
 ```
 
+- Arch:
+
+```
+sudo pacman -S python-pipx
+pipx ensurepath
+sudo pipx --global ensurepath # optional to allow pipx actions with --global argument
```

Comment:
I just opened #1397 for fixing it and `sudo pipx ensurepath --global` should work now.
```suggestion
sudo pipx ensurepath --global # optional to allow pipx actions with --global argument
```